lib/main.coffee

```diff
@@ -1,5 +1,4 @@
 {Notification, CompositeDisposable} = require 'atom'
-CommandLogger = require './command-logger'
```

Comment:
This package doesn't use activation commands so activate will occur right after load so I don't think it should be moved since it won't make a difference overall in the load plus activate time.